### DISCUSSION TITLE
Add app version number somewhere

### DISCUSSION THREAD

**Discussion by MadHoek** [majikat768/HuntStatsLogger#75] [Ideas]
In order to quickly check which version of the great HuntStatsLogger one currently uses it would be nice to check the version number.
I suggest to put it in the tilte bar like so: "Hunt Stats Logger v. x.xx"
Or, if you don't want it in the the title bar maybe put it in the settings window: The lower right corner would be a good place for "v. x.xx", probably.

**Comment by nbergman** [comment]
Agreed. I was just going to suggest this but glad MadHoek beat me to it.
